### PR TITLE
feat/recycle certs

### PR DESCRIPTION
**Pull request by pagottoo**
With this PR we can dump cert-manager resources (secrets/certs/clusterissuers) to recycle TLS/SSL.
You can run `go run . backupSSL`

### CODE REVIEW

File: cmd/backupSsl.go [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+	"github.com/kubefirst/kubefirst/internal/ssl"
+	"github.com/spf13/cobra"
+)
+
+// backupSslCmd represents the backupSsl command
+var backupSslCmd = &cobra.Command{
+	Use:   "backupSSL",
+	Short: "Backup Secrets (cert-manager/certificates) to bucket kubefirst-<DOMAIN>",
+	Long: `This command create a backupt of secrets from certmanager certificates to bucket named kubefirst-<DOMAIN>
```
Comment (6za) [inline cmd/backupSsl.go:15]: `k1-*`, right?

File: cmd/create.go [hunk 2]
```diff
@@ -106,16 +107,18 @@ to quickly create a Cobra application.`,
...
-		err = kPortForwardArgocd.Start()
-		defer kPortForwardArgocd.Process.Signal(syscall.SIGTERM)
-		if err != nil {
-			log.Printf("Commad Execution STDOUT: %s", kPortForwardArgocdOutb.String())
-			log.Printf("Commad Execution STDERR: %s", kPortForwardArgocdErrb.String())
-			log.Panicf("error: failed to port-forward to argocd in main thread %s", err)
+		if !dryRun {
+			var kPortForwardArgocdOutb, kPortForwardArgocdErrb bytes.Buffer
+			kPortForwardArgocd = exec.Command(config.KubectlClientPath, "--kubeconfig", config.KubeConfigPath, "-n", "argocd", "port-forward", "svc/argocd-server", "8080:80")
+			kPortForwardArgocd.Stdout = &kPortForwardArgocdOutb
```
Comment (6za) [inline cmd/create.go:113]: We need to double check if we are not breaking `main`here.. seems to be some old code getting a ride on create.

**Review by converge — APPROVED** [review]

**Comment by 6za** [comment]
This PR can be closed, and we can focus on #135  for review up to this point.

**Comment by pagottoo** [comment]
the work continue on #135

**Pull request closed without merging**